### PR TITLE
Add "how to submit" documentation.

### PR DESCRIPTION
**Pull request by mml**
Still need to link this into the main README.md someplace.

### CODE REVIEW

**Comment by mml** [comment]
@WilliamDenniss

**Comment by mml** [comment]
@timothysc

File: sonobuoy-conformance-1.7.yaml [hunk 1]
```diff
@@ -0,0 +1,194 @@
+---
```
Comment (timothysc) [inline sonobuoy-conformance-1.7.yaml]: were updating https://scanner.heptio.com/ for 1.8 right now.  Should have an update in about a week.

Comment (mml) [reply sonobuoy-conformance-1.7.yaml]: Cool.  Happy to update the instructions to refer to scanner whenever.  For now, I'm keeping it simple with this static yaml file.

File: instructions.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+See https://github.com/mml/k8s-conformance/pull/1 for a sample.
+
+## Amendment for Private Review
+
+If you need a private review... *TODO: @mml to describe the steps here*
+
+## Review
+
+A reviewer should be assigned to your PR shortly.  If you seek faster service,
+please contact ???@???.???.
```
Comment (WilliamDenniss) [inline instructions.md]: info@cncf.io

File: instructions.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+product_logo_url: https://yoyo.dyne/assets/turbo-encabulator.svg
+```
+
+### Sample PR
+
+See https://github.com/mml/k8s-conformance/pull/1 for a sample.
+
+## Amendment for Private Review
+
+If you need a private review... *TODO: @mml to describe the steps here*
```
Comment (WilliamDenniss) [inline instructions.md]: For now, let's say contact CNCF. info@cncf.io

**Pull request merged**